jobsdb/setup.go

```diff
@@ -0,0 +1,41 @@
+package jobsdb
+
+import (
+	"fmt"
+
+	migrator "github.com/rudderlabs/rudder-server/services/sql-migrator"
+)
+
+// setuDatabaseTables will initialize jobsdb tables using migration templates inside 'sql/migrations/jobsdb'.
+// Dataset tables are not created via migration scripts, they can only be updated.
+// The following data are passed to JobsDB migration templates:
+// - Prefix: The table prefix used by this jobsdb instance.
+// - Datasets: Array of existing dataset indices.
+func (jd *HandleT) setupDatabaseTables() {
```

Comment:
Let's call this DatasetTables?

Reply:
I named it like this because it is not strictly related to datasets, it is meant to also handle the rest of the tables, e.g journal.

Reply:
Calling it JobsDBTables might make more sense. Since there are going to be three calls for this once for each jobsdb?

Reply:
Well, its part of the JobsDB handle, and is called internally by its Setup method, so it is implied that it creates the database tables for this JobsDB instance. I can change it however, if you insist.

Reply:
Got it, this is fine.